source/enrollmentresponse/bundle-EnrollmentResponse-search-params.xml

```diff
@@ -59,4 +59,43 @@
       </SearchParameter>
     </resource>
   </entry>
+    <entry>
+    <resource>
+      <SearchParameter>
+        <id value="EnrollmentResponse-patient"/>
+        <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+          <valueCode value="trial-use"/>
+        </extension>
+        <extension url="http://hl7.org/fhir/build/StructureDefinition/path">
+          <valueString value="EnrollmentResponse.candidate"/>
+        </extension>
+        <url value="http://hl7.org/fhir/build/SearchParameter/EnrollmentResponse-patient"/>
+        <description value="The party to be enrolled"/>
+        <code value="patient"/>
+        <type value="reference"/>
+        <expression value="EnrollmentResponse.candidate"/>
+        <processingMode value="normal"/>
```

Comment:
Should specify target of Patient here so it's clear this is a reference to a Patient